test/utilities/FormManager.test.tsx

```diff
@@ -1,9 +1,15 @@
 import faker from 'faker';
+import httpStatus from 'http-status-codes';
 import * as Antd from 'antd';
 
 import { Tester } from '@mighty-justice/tester';
 
-import { FormCard, IFieldSetPartial } from '../../src';
+import { Form, FormCard, IFieldSetPartial } from '../../src';
+
+const ERROR_TYPES = [
```

Comment:
- I feel like `ERROR_TYPES` is a bit too vague. These are the errors we can get data out of. What's a good name for them? `ERRORS_WITH_CONTENT`? `FEEDBACK_STATUS_CODES`? `ERROR_WITH_DESCRIPTION`? or anything you can come up with.
- If you added this const to formManager, you could write `describeErrors = ERROR_TYPES.includes(status);` (or whatever the better name is) which is nice.
- You could also export it and import it here and then we'd only have it written once.